patches/api/0327-Add-PlayerCompostItemEvent.patch

```diff
@@ -0,0 +1,113 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Noah van der Aa <ndvdaa@gmail.com>
+Date: Sat, 7 Aug 2021 15:11:27 +0200
+Subject: [PATCH] Add PlayerCompostItemEvent.
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/event/entity/CompostItemEvent.java b/src/main/java/com/destroystokyo/paper/event/entity/CompostItemEvent.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..b30d155d4936b300c4e50ae9ff246bce8bd8fe05
+--- /dev/null
++++ b/src/main/java/com/destroystokyo/paper/event/entity/CompostItemEvent.java
+@@ -0,0 +1,101 @@
++package com.destroystokyo.paper.event.entity;
++
++import org.bukkit.block.Block;
++import org.bukkit.entity.Entity;
++import org.bukkit.event.Cancellable;
++import org.bukkit.event.HandlerList;
++import org.bukkit.event.entity.EntityEvent;
++import org.bukkit.inventory.ItemStack;
++import org.jetbrains.annotations.NotNull;
++import org.jetbrains.annotations.Nullable;
++
++/**
++ * Called when an item is about to be composted by an entity or a hopper.
++ */
++public class CompostItemEvent extends EntityEvent implements Cancellable {
```

Comment:
This can't extend `EntityEvent` because the entity is nullable.
Look at the super constructor you are calling, it's entity param is marked as not null.